tools/upgrader/util/upgrade.go

```diff
@@ -40,9 +40,9 @@ func initClient(kubeconfig string) error {
 	if kubeconfig == "" {
 		kubeconfig = util.GetDefaultKubeConfigPath()
 	}
-	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	config, err := clientcmd.NewRestConfigForDefaultSearchPath(kubeconfig)
 	if err != nil {
-		glog.Fatalf("BuildConfigFromFlags failed: %v", err)
+		glog.Fatalf("error creating rest config: %v", err)
```

Comment:
nit: A fatal log message should start with a capital letter since it will be the beginning of the log line that the user is reading (unlike error strings that bubble up the call stack).

Reply:
That makes sense, changing.